File: clients/Ruby/my_strategy.rb
require_relative 'model'

class MyStrategy
    def get_action(player_view, debug_interface)
        return Action.new(Hash.new)
    end
    def debug_update(player_view, debug_interface)
        debug_interface.send(DebugCommand::Clear.new())
        debug_interface.get_state()
    end
end

File: clients/Go/my_strategy.go
package main

import (
	. "aicup2020/model"
)

type MyStrategy struct{}

func NewMyStrategy() MyStrategy {
	return MyStrategy{}
}

func (strategy MyStrategy) getAction(playerView PlayerView, debugInterface *DebugInterface) Action {
	return Action{
		EntityActions: make(map[int32]EntityAction),
	}
}

func (strategy MyStrategy) debugUpdate(playerView PlayerView, debugInterface DebugInterface) {
	debugInterface.Send(DebugCommandClear{})
	debugInterface.GetState()
}
